resources/assets/js/annotations/ol/PolygonValidator.js

```diff
@@ -26,7 +31,7 @@ class PolygonValidator {
 
     getValidPolygon() {
         // Check if polygon is self-intersecting
-        const parser = new OL3Parser();
+        let parser = new OL3Parser();
```

Comment:
Why not let it be a `const`?

Reply:
I changed this file a lot, where the parser was initialized in the constructor at some point. When changing everything back, I forgot that this was a const.